lib/msf/ui/console/driver.rb

```diff
@@ -601,6 +605,15 @@ def handle_ssh_ident(val)
     $VERBOSE = verbose
   end
 
+  def handle_session_tlv_logging(val)
+    val = val.downcase
+    if val && (val == 'console' || val == 'false' || val =~ /^file:.+/)
```

Comment:
Is it worth validating the file path exists here? 🤔 

There's still the potential of this path being deleted after the validation occurs though, so you'll still want error handling later inside the packet dispatcher

Reply:
not a blocker, similar to the conversation we talked about before moving the default file name logic here might be better:
```
setg sessiontlvlogging file:/tmp
```

So later anyone that wants to view the value of the sessiontlvlogging path, it'll show the actual file it will be written to:
```
setg
=> sessiontlvlogging = file:/tmp/sessiontlvlogging.txt
```